core/templates/dev/head/services/context.service.ts

```diff
@@ -225,6 +225,17 @@ export class ContextService {
       this.getPageContext() ===
         ServicesConstants.PAGE_CONTEXT.EXPLORATION_EDITOR);
   }
+
+  // Following method helps to know whether RTE components requiring fullscreen
+  // can be used in a particular mode.
+  canWriteToFs(): boolean {
```

Comment:
Please add unit tests for this function.

Reply:
yeah, I'll add them.